wpimath/src/main/java/edu/wpi/first/math/Matrix.java

```diff
@@ -289,6 +289,26 @@ public Matrix<R, C> div(double value) {
     return new Matrix<>(this.m_storage.divide(value));
   }
 
+  /**
+   * Solves the least-squares problem Ax=B using a QR decomposition with full pivoting, where this
+   * matrix is A.
+   *
+   * @param other The B matrix.
+   * @return The solution matrix.
+   */
+  public final <R2 extends Num, C2 extends Num> Matrix<C, C2> mldivide(Matrix<R2, C2> other) {
```

Comment:
What does mldivide mean?

Reply:
it's in reference to https://www.mathworks.com/help/matlab/ref/mldivide.html, open to a better name though

Reply:
Matrix.solve() already does this, unless you need the full-pivot QR decomp specifically, or the EJML impl is too slow.

Reply:
Yeah, I was running into convergence test failures with EJML's col-pivot QR. The full-pivot QR seems to reliably converge to within 1m of the final pose in the convergence test where the col-pivot QR wouldn't.

Reply:
In that case, we should document that solve() uses column-pivoting. After digging through https://ejml.org/javadoc/org/ejml/dense/row/decomposition/qr/package-summary.html, all I could find is the column-pivoting impl.

As for the name of _this_ function, I'd prefer that it matched the JNI name so the C++ and Java uses in the UKF look similar.